department-of-veterans-affairs/caseflow
# Upload page

Connects #5819 

### Description
This PR adds the UI for the upload page of the Hearing Schedule flow. Selecting the date range, uploading the file, and clicking 'continue' will be handled in future PRs. Also, something is broken on the loading page, so props are getting entirely reloaded every page change. That will be addressed in a future PR. (#6076)

### Testing Plan
1. Go to /hearings/schedule/build, click 'Upload files', and confirm the page loads correctly
1. Refresh the page and confirm the page loads correctly again
